native/acter/api.rsh

```diff
@@ -2543,14 +2549,20 @@ object Invitation {
     /// get the room id of this invitation
     fn room_id() -> RoomId;
 
-    /// get the room name of this invitation
-    fn room_name() -> Future<Result<string>>;
+    /// the RoomId as a String
+    fn room_id_str() -> string;
+
+    /// get the room of this invitation
+    fn room() -> Room;
 
     /// get the user id of this invitation sender
-    fn sender() -> UserId;
+    fn sender_id() -> UserId;
```

Comment:
Same goes for this ```UserId```. We only seem to use its string version which is readable and processable to flutter...